app/src/main/java/com/mashup/ui/model/SNSModel.kt

```diff
@@ -0,0 +1,7 @@
+package com.mashup.ui.model
+
+data class SNSModel(
+    val name: String,
+    val iconRes: Int,
+    val link: String,
+)
```

Comment:
```suggestion
data class SNSModel(
    val name: String,
    @DrawableRes val iconRes: Int,
    val link: String,
)
```

개인적으로 저는 Resource를 사용하는 곳에 어노테이션을 붙이고 있어요! @StringRes, @DrwableRes, @ColorRes

매개변수 주입 시점에 IDE가 검증을 해주더라구요! 참고 해주세요 :)
![image](https://user-images.githubusercontent.com/33657541/208306531-d64e5b2c-acb5-4809-8ffc-8cac54178eae.png)